dispatcher/backend/src/common/schemas/fields.py

```diff
@@ -75,3 +75,10 @@ def validate_multiple_of_100(value):
 offliner_field = String(required=False, validate=validate_offliner)
 email_field = fields.Email(required=False, validate=validate_not_empty)
 username_field = String(required=True, validate=validate_not_empty)
+
+
+def validate_sort_order(value):
```

Comment:
No need to reimplement that, just use `validate.OneOf(...)` (there are plenty other examples in this file)